src/content.js

```diff
@@ -23,6 +23,8 @@ const colors = {
   "Международный гроссмейстер": "#ff0000",
   "Legendary Grandmaster": "#ff0000",
   "Легендарный гроссмейстер": "#ff0000",
+  Tourist: "#ff0000",
```

Comment:
The tourist title has only the first letter as red, the other letters are black. See the code to handle legendary grandmasters. You have to code that part too. The highlight user row works good.